### DISCUSSION TITLE
Information about Aurulent

### DISCUSSION THREAD

**Discussion by tasty0tomato** [ryanoasis/nerd-fonts#797] [Q&A]
I love Aurulent very much but there is not enough information about it. Can someone tell me something about it, please?

**Comment by Finii** [comment]
Well, it has been designed by [Stephen G. Hartke (search engine results link)](https://duckduckgo.com/?q=Stephen+G.+Hartke) in 2006-2007.

Other fonts designed by him:
https://www.dafont.com/stephen-g-hartke.d1583

I do not know of a current website, but this was where he published the font back then:
https://web.archive.org/web/20101117152634/http://delubrum.org:80/

Here some [insights](https://web.archive.org/web/20080917171840/http://typophile.com/node/31970) in the development process (link taken from delubrum.org).

Geocities were the death of many a website :-(
Here with the beginning of Aurulent: https://web.archive.org/web/20061108024144/http://www.geocities.com:80/hartke01/
The next website is identical to the one on delubrum.

Original release package for example here: https://www.samiam.org/fonts/other/AurulentSans-20070504.tgz
Or all files extracted for you convenience here: https://github.com/deepfire/hartke-aurulent-sans

_Edit: Ah, found it on geocities, adding links; add release package link_
